### DISCUSSION TITLE
Action to release the ladder in platformer object extension

### DISCUSSION THREAD

**Discussion by shadow00dev** [4ian/GDevelop#2339] [Archive]
## Description
Real simple request, a new platformer control/option that allows the player to 'Drop or Let Go' of a ladder.
This would allow us to de-attach the player whilst on the ladder for various reasons, in my case when they're at the bottom and on solid ground.

## Solution suggested
Describe the solution
When activated, the player would let go of the ladder and set isOnLadder() to false. If the player is in the air when activated, they would fall to the ground (much like when you jump excluding the going up part).

I tried doing this is JS, but it looks like nothing is exposed to let me do this.

**Comment by Silver-Streak** [comment]
Isn't that just simulate jump? I don't think there is an actual "release ladder" action in the game to simulate.

**Comment by 4ian** [comment]
Added on the roadmap. See other improvements to the platformer engine that can be done on the roadmap: https://trello.com/b/qf0lM7k8/gdevelop-roadmap?menu=filter&filter=label:Feat%3A%20platformer%20engine

**Comment by arthuro555** [comment]
Done in https://github.com/4ian/GDevelop/commit/a1b840a4fac208c9db24e4fa9c67ccea99ebfb4a by @ClementPasteau
